openhands/runtime/utils/runtime_build.py

```diff
@@ -41,7 +41,9 @@ def _create_project_source_dist():
     logger.info(f'Using project root: {project_root}')
 
     # run "python -m build -s" on project_root to create project tarball
-    result = subprocess.run(f'python -m build -s {project_root}', shell=True)
+    result = subprocess.run(
+        f'python -m build -s ' + project_root.replace(" ", r"\ "), shell=True
+    )
```

Comment:
I had to remove the f-string here because the build server is over 1 year old and does not allow f strings that contain escape characters (`\`)